algorithms/facenet/vectorize.py

```diff
@@ -13,17 +13,24 @@
 Image = np.array
 
 
-def get_face_vectors(img_path: str, prealigned: bool) -> List[FaceVector]:
+def get_face_vectors_batch(
+        img_paths: List[str], prealigned: bool) -> List[List[FaceVector]]:
     identifier = face.Identifier(
         facenet_model_checkpoint='20180402-114759.pb')
-    image = identifier.get_image_from_path(img_path)
-    vectors = identifier.vectorize(image, prealigned=prealigned)
-    return [vector.tolist() for vector in vectors]
+
+    images = map(identifier.get_image_from_path, img_paths)
+    all_vectors = identifier.vectorize_all(images, prealigned=prealigned)
+    np_to_list = []
+    for vectors in all_vectors:
+        np_to_list.append([vector.tolist() for vector in vectors])
+    return np_to_list
```

Comment:
Consider whether the list comprehension version of this would be more readable:

```py
return [[vector.tolist() for vector in vectors] for vector in all_vectors]
```